ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -402,120 +402,132 @@
+        date1-last-stop (last (:stoptimes (first (:selected-trip-pair compare))))
+        date2-first-stop (first (:stoptimes (second (:selected-trip-pair compare))))
+        date2-last-stop (last (:stoptimes (second (:selected-trip-pair compare))))
+        date1-label (str ", " (time/format-interval-as-time (:gtfs/departure-time date1-first-stop)) " " (:gtfs/stop-name date1-first-stop) " - "
+                         (time/format-interval-as-time (:gtfs/departure-time date1-last-stop)) " " (:gtfs/stop-name date1-last-stop))
+        date2-label (str ", " (time/format-interval-as-time (:gtfs/departure-time date2-first-stop))" " (:gtfs/stop-name date2-first-stop) " - "
+             (time/format-interval-as-time (:gtfs/departure-time date2-last-stop)) " " (:gtfs/stop-name date2-last-stop))]
+    [tv-utilities/section {:open? (get open-sections :trip-stop-sequence true)
+                           :toggle! #(e! (tv/->ToggleSection :trip-stop-sequence))}
+     "Pysäkit"
+     "Pysäkkilistalla näytetään valitun vuoron pysäkkikohtaiset aikataulut."
+     (let [second-stops-empty? (empty? (:stoptimes (second selected-trip-pair)))]
+       [:div
+        [tv-utilities/date-comparison-icons-with-date-labels compare date1-label date2-label true]
+        [:div.trip-stop-sequence {:style {:margin-top "1em"}}
+         [table/table {:name->label str
+                       :label-style style-base/table-col-style-wrap}
+          [{:name "Pysäkki"
+            :read :gtfs/stop-name
+            :col-style style-base/table-col-style-wrap
+            :format (partial format-stop-name)}
+           {:name "Lähtöaika"
+            :read :gtfs/departure-time-date1
+            :col-style style-base/table-col-style-wrap
+            :format (partial format-stop-time (style/date1-highlight-style))}
+           {:name "Lähtöaika"
+            :read :gtfs/departure-time-date2
+            :col-style style-base/table-col-style-wrap
+            :format (partial format-stop-time (style/date2-highlight-style))}
+           {:name "Muutokset"
+            :read identity
+            :col-style style-base/table-col-style-wrap
+            :format (fn [{:gtfs/keys [departure-time-date1 departure-time-date2]}]
+                      (cond
+                        (and departure-time-date1 (nil? departure-time-date2))
+                        (if second-stops-empty? "Poistuva vuoro" "Pysäkki ei kuulu reitille")
+
+                        (and (nil? departure-time-date1) departure-time-date2)
+                        "Uusi pysäkki reitillä"
+
+                        (not= departure-time-date1 departure-time-date2)
+                        [icon-l/icon-labeled [ic/action-query-builder]
+                         (time/format-minutes-elapsed
+                           (time/minutes-elapsed departure-time-date1 departure-time-date2))]
+
+                        :default
+                        [icon-l/icon-labeled {:style {:color "lightgray"}}
```

Comment:
Ok, but a good practise would be to define a logical symbol for a style/color and then define it in `colors.cljs`.
That way color/theme changes in future can be done simply by versioning/editing `colors.cljs` without changing any other source code for views.

Reply:
Yes. My bad. Changed now